### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by chrishajduk84**
Fixed and updated build information in Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,27 +18,51 @@ CONOPS/Rules:
...
+1. Install the dependencies listed above. This will vary based on your system. 
+2. Clone the Project
+
+   `git clone https://github.com/UWARG/computer-vision.git`
+
+   `cd computer-vision`
+
 3. Use CMake to setup the project from the CMake list
+
+   `mkdir build`
```
Comment (chrishajduk84) [inline README.md]: Made concise. Build script and build script options need to be written into the docs.uwarg.com website.

File: README.md [hunk 2]
```diff
@@ -16,29 +16,43 @@ Photos to use for testing:
...
 
 ####Installation Instructions:
-1. Clone the Project
-2. Install dependencies
+1. Install the dependencies listed above. This will vary based on your system. 
+2. Clone the Project
 3. Use CMake to setup the project from the CMake list
 4. Compile
 
+For Linux, refer to the detailed instructions here: [Linux Installation](http://docs.uwarg.com/computer-vision/Building-the-project-[Linux]/)
```
Comment (benjaminwinger) [inline README.md]: If we're going to be just referencing the detailed instructions, I don't think we also need the list above. Also, on a Windows system there's little use in installing the dependencies, cloning the project running CMake and then using a docker image instead.

You could put the Docker info before the instruction list, keep the list, and squash the last two instructions to a single `./build.sh` command with a reference to the docs for more detailed information and build options, or just get rid of the list altogether and just reference the docs.

Comment (chrishajduk84) [inline README.md]: Ok. I cleaned it up a bit more.
I combined 3.+4., but honestly the list is so short, I don't know if I should just remove it.

File: README.md [hunk 3]
```diff
@@ -16,33 +16,47 @@ Photos to use for testing:
...
 ####Goal:
 To fill this chart with the right values:
 [Judges Evaluation Sheet](https://drive.google.com/open?id=0B8ozhZojJMQbWTBWaXBiXzBQcFk)
 
-This includes latitudes, longitudes, centroids, areas, volumes, and QR codes.
+This evaluation sheet includes spots to fill with latitudes, longitudes, centroids, areas, volumes, and QR code values.
 
 ### Licensing
-Copyright (c) 2015, Waterloo Aerial Robotics Group (WARG)
+Copyright (c) 2016, Waterloo Aerial Robotics Group (WARG)
```
Comment (benjaminwinger) [inline README.md]: Shouldn't this be 2015-2016?

Comment (chrishajduk84) [inline README.md]: Yes it should.

**Pull request merged**